src/KawaBot.Tests/BasicControllerTests.fs

```diff
@@ -16,12 +19,15 @@ type BasicControllerTests =
             _server = new Http.HttpListener("http://*:80/")
             _feeder = new Feeder(80)
         } then
+            let config = new LoggingConfiguration()
+            config.AddRule(LogLevel.Trace, LogLevel.Fatal, new ConsoleTarget(), "*")
+            LogManager.Configuration <- config
```

Comment:
Can't we put this configuration into the `App.config` file? And moreover with applying for all loggers?